### PR TITLE
Send a conflict response instead of bad request when model is already registered

### PR DESCRIPTION
**Pull request by alexwong**
## Issue #, if available:
None

## Description of changes:
Change response when attempting to register a model that has already been registered.

## Testing done:
Tested locally. Note CI tests not running right now (run_ci_tests.sh doesn't seem to run anything).

**To run CI tests on your changes refer [README.md](https://github.com/awslabs/mxnet-model-server/blob/master/ci/README.md)**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: frontend/server/src/main/java/com/amazonaws/ml/mms/http/HttpRequestHandler.java [hunk 1]
```diff
@@ -58,6 +58,9 @@ protected void channelRead0(ChannelHandlerContext ctx, FullHttpRequest req) {
         } catch (BadRequestException | ModelException e) {
             logger.trace("", e);
             NettyUtils.sendError(ctx, HttpResponseStatus.BAD_REQUEST, e);
+        } catch (ConflictStatusException e) {
```
Comment (vdantu) [inline frontend/server/src/main/java/com/amazonaws/ml/mms/http/HttpRequestHandler.java]: You might want to add a test as well [ModelServerTest.java](https://github.com/awslabs/mxnet-model-server/blob/master/frontend/server/src/test/java/com/amazonaws/ml/mms/ModelServerTest.java). Also, update openapi spec ([management_open_api.json](https://github.com/awslabs/mxnet-model-server/blob/master/frontend/server/src/test/resources/management_open_api.json#L825) and [OpenApiUtils.java](https://github.com/awslabs/mxnet-model-server/blob/master/frontend/server/src/main/java/com/amazonaws/ml/mms/openapi/OpenApiUtils.java#L387)) to reflect this change.

**Comment by vdantu** [comment]
@alexwong : Welcome to the contributors club 👍

File: frontend/server/src/main/java/com/amazonaws/ml/mms/http/HttpRequestHandler.java (hunk 1 above)
Comment (alexwong) [reply frontend/server/src/main/java/com/amazonaws/ml/mms/http/HttpRequestHandler.java]: Will do, thanks!

**Comment by vdantu** [comment]
Looks good to me.. Thanks for addressing the comments.

**Pull request merged**